lib/web_console/exception_mapper.rb

```diff
@@ -1,8 +1,11 @@
 # frozen_string_literal: true
+require 'delegate'
 
 module WebConsole
-  class ExceptionMapper
+  class ExceptionMapper < SimpleDelegator
     def initialize(exception)
+      super(exception)
```

Comment:
Can we have this as an `attr_reader :exc` instead of the need of the whole delegation? I think we need to access it only to find the exception by its object ID. If we hide the find operation behind a class method, the attribute can even be `protected`.